src/domain/mainwindow.py

```diff
@@ -44,10 +46,10 @@ def __init__(self):
         self.cameras_list = []
 
         # Connect Actions
-        self._connect_actions()
+        self.__connect_actions()
```

Comment:
Why are you using the Python's name mangling (prefix with double underscore)? It's better to avoid, unless it's really necessary.

Same below, in all other cases.